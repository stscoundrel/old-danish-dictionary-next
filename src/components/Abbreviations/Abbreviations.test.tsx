import React from 'react'
import renderer from 'react-test-renderer'
import Abbreviations from './index'

describe('Abbreviations component', () => {
  const abbreviations = [
    { abbreviation: 'isl.', explanation: 'islandsk.' },
    { abbreviation: 'n.', explanation: 'norsk.' },
    { abbreviation: 'no.', explanation: 'navneord (substantivum).' },
    { abbreviation: 'æ.', explanation: 'ældre.' },
  ]

  test('Matches the snapshot', () => {
    const tree = renderer.create(
      <Abbreviations abbreviations={abbreviations} />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
