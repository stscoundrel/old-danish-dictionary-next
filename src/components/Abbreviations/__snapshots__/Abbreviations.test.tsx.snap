// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Abbreviations component Matches the snapshot 1`] = `
<div
  className="abbreviations"
>
  <div
    className="column"
  >
    <h4>
      Abbreviations used:
    </h4>
    <dl
      className="wrap"
    >
      <dt
        className="abbreviation"
      >
        <strong>
          isl.
        </strong>
      </dt>
      <dd
        className="explanation"
      >
        islandsk.
      </dd>
    </dl>
    <dl
      className="wrap"
    >
      <dt
        className="abbreviation"
      >
        <strong>
          n.
        </strong>
      </dt>
      <dd
        className="explanation"
      >
        norsk.
      </dd>
    </dl>
    <dl
      className="wrap"
    >
      <dt
        className="abbreviation"
      >
        <strong>
          no.
        </strong>
      </dt>
      <dd
        className="explanation"
      >
        navneord (substantivum).
      </dd>
    </dl>
    <dl
      className="wrap"
    >
      <dt
        className="abbreviation"
      >
        <strong>
          æ.
        </strong>
      </dt>
      <dd
        className="explanation"
      >
        ældre.
      </dd>
    </dl>
  </div>
</div>
`;
